run--03b--postestimation-disaggregate-estimates.Rmd

```diff
@@ -350,15 +350,21 @@ add_agg_level <- function(ests, agg_level) {
         geo_crosswalk %>% 
         transmute(GEOID    = GEOID, 
                   County   = County, 
-                  geo_part = 1) %>% 
+                  geo_part = pct_area) %>% 
         unique()
     } else if (agg_level == "aux_geo_label") {
       geo_merge_data <- 
         geo_crosswalk_aux %>% 
         transmute(GEOID         = GEOID, 
                   aux_geo_label = aux_geo_label, 
                   geo_part      = pct_area)
-    }  
+    } else if (my_state_abbr == "IL") {
```

Comment:
As above -- this shouldn't be in `main`. We should take time to expand the `aux` specification.

(I won't make this comment to every instance in later files, but it does apply.)